### ISSUE TITLE
recommended variables in `MultinomialOutcomeTreatment`

### ISSUE THREAD

**Issue by bkj** [WinVector/pyvtreat#6]
I noticed that if you use `MultinomialOutcomeTreatment.fit_transform`  and then follow the recipe in the examples of selecting variables with
```
good_variables = plan.score_frame_.variable[plan.score_frame_.recommended].values
```
you'll end up with duplicated entries in `good_variables` (because you're doing separate tests for each output class, I think?)

Would you suggest just calling
```
good_variables = plan.score_frame_.variable[plan.score_frame_.recommended].unique()
```
instead?  Or is there some other recommended way to do variable selection in the `MultinomialOutcomeTreatment` case?

Thanks!
~ Ben

Snippet:
```
import vtreat
import pandas as pd
from sklearn.datasets import load_iris

iris = load_iris()

X, y = pd.DataFrame(iris['data']), iris['target']

plan  = vtreat.BinomialOutcomeTreatment(outcome_target=True)
X_new = plan.fit_transform(X, y == 0)
plan.score_frame_

#    variable  orig_variable   treatment  y_aware  has_range  PearsonR  significance  vcount  default_threshold  recommended
# 0         0              0  clean_copy    False       True  0.079396  3.341524e-01     4.0               0.25        False
# 1         1              1  clean_copy    False       True -0.467703  1.595624e-09     4.0               0.25         True
# 2         2              2  clean_copy    False       True  0.201754  1.329302e-02     4.0               0.25         True
# 3         3              3  clean_copy    False       True  0.117899  1.507473e-01     4.0               0.25         True

plan  = vtreat.MultinomialOutcomeTreatment()
X_new = plan.fit_transform(X, y)
plan.score_frame_
#     variable  orig_variable   treatment  y_aware  has_range  PearsonR  significance  vcount  default_threshold  recommended  outcome_target
# 0          0              0  clean_copy    False       True -0.717416  5.288768e-25     4.0               0.25         True               0
# 1          1              1  clean_copy    False       True  0.603348  3.054699e-16     4.0               0.25         True               0
# 2          2              2  clean_copy    False       True -0.922765  3.623379e-63     4.0               0.25         True               0
# 3          3              3  clean_copy    False       True -0.887344  1.288504e-51     4.0               0.25         True               0
# 4          0              0  clean_copy    False       True  0.079396  3.341524e-01     4.0               0.25        False               1
# 5          1              1  clean_copy    False       True -0.467703  1.595624e-09     4.0               0.25         True               1
# 6          2              2  clean_copy    False       True  0.201754  1.329302e-02     4.0               0.25         True               1
# 7          3              3  clean_copy    False       True  0.117899  1.507473e-01     4.0               0.25         True               1
# 8          0              0  clean_copy    False       True  0.638020  1.619533e-18     4.0               0.25         True               2
# 9          1              1  clean_copy    False       True -0.135645  9.791170e-02     4.0               0.25         True               2
# 10         2              2  clean_copy    False       True  0.721011  2.381987e-25     4.0               0.25         True               2
# 11         3              3  clean_copy    False       True  0.769445  1.297773e-30     4.0               0.25         True               2
```

**Comment by JohnMount** [comment]
Very good point, sorry about that and thank you for the correction. I will update the example to de-dup the original variable names as you point out.

**Comment by JohnMount** [comment]
Ben, thanks again for your help.  

I have added some notes to the [multinomial example](https://github.com/WinVector/pyvtreat/blob/master/Examples/Multinomial/MultinomialExample.md) discussing proper variable selection in this case.  I have also added some notes to your [iris example](https://github.com/WinVector/pyvtreat/blob/master/Examples/Multinomial/iris.md) along related lines.  

The multinomial path is a bit more exotic than the others, so we haven't yet documented it as well as the other methods.  I'll see if I can get Nina Zumel to get some time (not sure when) to flesh out the multinomial classification example into something nice (as she did for the regression, binomial classification, and unsupervised cases).
